src/dso_api/dynamic_api/routers.py

```diff
@@ -264,6 +265,13 @@ def _prune_app_registry(self, old_dynamic_apps: set):
             del apps.all_models[removed_app]
 
     def filter_datasets(self, queryset):
+        """Filter datasets:
+        - remove Non-default datasets
+        - include only datasets defined in DATASETS_LIST (if settings.DATASETS_LIST is defined)
+        - exclude any datasets in DATASETS_EXCLUDE list (if settings.DATASETS_EXCLUDE is defined)
+        """
+        queryset = queryset.filter(Q(version=None) | Q(is_default_version=True))
```

Comment:
Can we add a manager/queryset method to schematools: `queryset.default_version()`?

Reply:
This is only place where default version filtering is applied. And queryset/manager is a bit of overkill.